apps/web/src/components/tx-flow/TxInfoProvider.tsx

```diff
@@ -15,7 +15,7 @@ type SimulationStatus = {
 
 const initialSimulation: UseSimulationReturn = {
   simulateTransaction: () => {},
-  simulation: undefined,
+  simulationData: undefined,
```

Comment:
Please remove any renaming that is not necessary for this PR. If you want to rename code to improve readibility, put all the renaming in a single commit.